resource/default/Mac/IntelliJ.xml

```diff
@@ -1,4 +1,4 @@
-<keymap version="1" name="macOS" parent="Mac OS X 10.5+">
+<keymap version="1" name="macOS" parent="Mac OS X 10.5+[Quote]>
```

Comment:
Hey @kasecato, looks like some treacherous `[Quote]` sneaked in 🙈

Noticed cause my imports were failing 
![image](https://github.com/kasecato/vscode-intellij-idea-keybindings/assets/697301/71eccc91-aafa-40d4-8dbe-6318efaf4c39)